CHANGELOG.md

```diff
@@ -32,6 +33,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add `toolbarBehavior` for `Toolbar` component and apply `buttonBehavior` for `ToolbarItem` component @sophieH29 ([#1468](https://github.com/stardust-ui/react/pull/1468))
 - Integrate ARIA HTML design pattern in the `Tree` component @silviuavram ([#1488](https://github.com/stardust-ui/react/pull/1488))
 - Add 'broadcast' icon to Teams theme @lawrencecushman ([#1509](https://github.com/stardust-ui/react/pull/1509))
+- Add `disabled` prop and `colorDisabled`, `backgroundColorDisabled`, `borderColorDisabled`, `borderStyle`, `borderWidth` variables for `Segment` component @Bugaa92 ([#1516](https://github.com/stardust-ui/react/pull/1516))
```

Comment:
```suggestion
- Add `disabled` prop for `Segment` component @Bugaa92 ([#1516](https://github.com/stardust-ui/react/pull/1516))
```

Not sure that we should mention each variable in changelog until there are no breaking changes

Reply:
agreed, applied changes